packages/shared/src/contexts/SettingsContext.tsx

```diff
@@ -94,7 +94,7 @@ const defaultSettings: RemoteSettings = {
   showTopSites: true,
   sidebarExpanded: true,
   sortingEnabled: false,
-  showWeeklyGoals: true,
+  optOutWeeklyGoal: true,
```

Comment:
The default value here now should be `false` since we want to initially include everyone.
```suggestion
  optOutWeeklyGoal: false,
```

Reply:
Nice spot, thanks Lee.